x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/command_and_control_rdp_remote_desktop_protocol_from_the_internet.json

```diff
@@ -76,5 +76,5 @@
   "timeline_title": "Comprehensive Network Timeline",
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 12
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950